webapp/src/components/cardDetail/attachment.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import {useIntl} from 'react-intl'
+
+import FileElement from '../../components/content/fileElement'
+import {ContentBlock} from '../../blocks/contentBlock'
+
+import './attachment.scss'
+import {Block} from '../../blocks/block'
+import CompassIcon from '../../widgets/icons/compassIcon'
+
+type Props = {
+    count: number
+    contents: ContentBlock[]
+    onDelete: (block: Block) => void
+    addAttachment: () => void
+}
+
+const Attachment = (props: Props): JSX.Element|null => {
```

Comment:
`null` is never returned...

Reply:
Also, since this encompasses all "Attachments" it should be named "AttachmentsList" following "CommentsList" as example.

Reply:
Got it.